Mlem/App/Utility/Extensions/Content Models/Comment1Providing+Extensions.swift

```diff
@@ -32,7 +32,9 @@ extension Comment1Providing {
             leadingActions: {
                 if api.canInteract {
                     upvoteAction(feedback: [.haptic])
-                    downvoteAction(feedback: [.haptic])
+                    if api.myInstance?.downvotesEnabled ?? true {
```

Comment:
*nit*

This `api.myInstance?.downvotesEnabled ?? true` phrase gets repeated a lot--it would be cleaner to have it as a computed var `api.downvotesEnabled`.